dev/com.ibm.ws.wsat.common/bnd.bnd

```diff
@@ -1,5 +1,5 @@
 #*******************************************************************************
-# Copyright (c) 2019,2021 IBM Corporation and others.
+# Copyright (c) 2019, 2022 IBM Corporation and others.
```

Comment:
Shouldn't com.ibm.ws.wsat.common.3.2 get deleted as well?

Reply:
Grr, don't know how this snuck back in here. I'd deleted it. Let me do it again.